opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/__init__.py

```diff
@@ -223,29 +268,37 @@ def create_histogram(self, name, unit="", description="") -> APIHistogram:
             self._measurement_consumer,
             unit,
             description,
+            advisory,
         )
         with self._instrument_id_instrument_lock:
             self._instrument_id_instrument[instrument_id] = instrument
             return instrument
 
-    def create_gauge(self, name, unit="", description="") -> APIGauge:
+    def create_gauge(
+        self, name, unit="", description="", advisory=None
+    ) -> APIGauge:
         (
-            is_instrument_registered,
             instrument_id,
-        ) = self._is_instrument_registered(name, _Gauge, unit, description)
+            status,
+        ) = self._register_instrument(
+            name, _Gauge, unit, description, advisory
+        )
 
-        if is_instrument_registered:
+        if status.conflict:
```

Comment:
I think there might be a logging bug here for the instruments with the placeholder `advisory=None`. Since we always use None, any duplicate registration (even non-conflicting) will show as conflicting `None == None` and log out here.

Reply:
"None == None" is True here so don't expect any issue